media/videostitcher/create_slate_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+package videostitcher
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+
+	stitcher "cloud.google.com/go/video/stitcher/apiv1"
+	stitcherstreampb "cloud.google.com/go/video/stitcher/apiv1/stitcherpb"
+)
+
+func TestCreateSlate(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := &bytes.Buffer{}
+
+	uuid, err := getUUID()
+	if err != nil {
+		t.Fatalf("getUUID err: %v", err)
+	}
+	slateID := fmt.Sprintf("%s-%s", slateID, uuid)
+	slateName := fmt.Sprintf("projects/%s/locations/%s/slates/%s", tc.ProjectID, location, slateID)
+	testutil.Retry(t, 3, 2*time.Second, func(r *testutil.R) {
+		if err := createSlate(buf, tc.ProjectID, slateID, slateURI); err != nil {
+			t.Fatalf("createSlate got err: %v", err)
+		}
+		if got := buf.String(); !strings.Contains(got, slateName) {
+			t.Fatalf("createSlate got\n----\n%v\n----\nWant to contain:\n----\n%v\n----\n", got, slateName)
+		}
+	})
+	buf.Reset()
+	teardownTestCreateSlate(slateName, t)
```

Comment:
question: are there any code paths where a slate is created but this code isn't called? Consider whether you can use `t.Cleanup()` to ensure that resources are torn down.

Reply:
Not sure what you mean. There should be no code paths where a slate is created and teardownTestCreateSlate() does not need to be called. It always needs to be called to clean up the slate.

Reply:
Not "doesn't need to be called" but "won't be called." In other words, is there a failure state where resources aren't cleaned up?

This isn't a blocking issue, so we can discuss over chat.